content/swift/concepts/dictionaries/terms/values/values.md

```diff
@@ -0,0 +1,44 @@
+---
+Title: '.values'
+Description: '.value is used to loop through a collection of values within a dictionary.'
+Subjects:
+  - 'Mobile Development'
+  - 'iOS'
+  - 'Computer Science'
+Tags:
+  - 'Dictionary'
+  - 'Properties'
+  - 'Methods'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **`.Values`** method is used to loop through a stored collection of values within a key-value dictionary. 
+
+## Syntax
+
+```pseudo
+print(dictionaryName.values)
+```
+- `dictionaryName` is the argument for the name of the dictionary you are wanting to pull the values from.
+- `.value` is used to call the collection of values within the dictionary named within the argument.
```

Comment:
```suggestion

- `dictionaryName`: Refers to the dictionary instance from which the values are accessed using the `.values` property.
```

Reply:
Avoid any "you", "we" and also, `dictionaryName` is not an argument here, it is rather the instance of the dictionary